net4j/com.b2international.snowowl.eventbus/src/com/b2international/snowowl/internal/eventbus/EventBus.java

```diff
@@ -227,6 +281,21 @@ public ExecutorService getExecutorService() {
 		return executorService;
 	}
 	
+	@Override
+	public long getInQueueMessages(String tag) {
+		return getOrCreateCounter(tag, inQueueMessages).get();
+	}
+
+	@Override
+	public long getProcessingMessages(String tag) {
+		return getOrCreateCounter(tag, currentlyProcessingMessages).get();
+	}
+	
+	@Override
+	public long getFinishedMessages(String tag) {
```

Comment:
How about adding a `getFailedMessages(String tag)` method, which would return the number of failed messages so far?